RELEASE.md

```diff
@@ -31,6 +38,10 @@ Many thanks to the following Kedroids for contributing PRs to this release:
 
 * [Jason Hite](https://github.com/jasonmhite)
 * [IngerMathilde](https://github.com/IngerMathilde)
+* [Laíza Milena Scheid Parizotto](https://github.com/laizaparizotto)
+* [Richard](https://github.com/CF-FHB-X)
+* [flpvvvv](https://github.com/flpvvvv)
+* [qheuristics](https://github.com/qheuristics)
```

Comment:
There's also another contribution from @/miguel-ortiz-marin in this release

Reply:
Thanks, yes, there are probably more. I only looked at the docs subfolder for third party contributors and added them (if you recall, they were in a separate section of the release notes). So this is not a comprehensive list of community contributors but it _is_ comprehensive for docs (up to today at least 17/10)

Reply:
Yep, that makes sense, I thought it'd be easier to add it in this PR since there was only one more.

Reply:
I'll do another check for contributors on the release PR!